ruleset/sca/amazon/cis_amazon_linux_2.yml

```diff
@@ -133,10 +101,10 @@ checks:
     condition: any
     rules:
       - 'c:mount -> r:\s/tmp\s'
-      - 'c:systemctl is-enabled tmp.mount -> r:enabled'
+      - 'p:tmp.mount'
```

Comment:
The CIS benchmark doesn't check for the process `tmp.mount`. Check that the title says `Ensure /tmp is configured`, so we would have to apply this check:
```
# systemctl show "tmp.mount" | grep -i unitfilestate
```